packages/server/src/utils/setupTaskService.ts

```diff
@@ -83,6 +83,13 @@ export default async function(env: Env, models: Models, config: Config, services
 			schedule: '*/15 * * * *',
 			run: (models: Models) => models.user().deleteExpiredAuthCodes(),
 		},
+
+		{
+			id: TaskId.DeleteArchivedBackups,
+			description: taskIdToLabel(TaskId.DeleteArchivedBackups),
+			schedule: '0 0 */2 * *',
```

Comment:
Let's change it to once a day please so that it more strictly matches the cut off date